src/eval.h

```diff
@@ -80,10 +80,8 @@ class Frame {
 class ScopedFrame {
  public:
   ScopedFrame(Evaluator* ev, Frame* frame);
-  // We only allow moving; copying would double stack frames
-  ScopedFrame(const ScopedFrame& other) = delete;
-  ScopedFrame& operator=(const ScopedFrame&) = delete;
-  ScopedFrame(ScopedFrame&& other);
+  // We disallow any copies or moves of this object.
+  ScopedFrame(ScopedFrame&& other) = delete;
```

Comment:
Can we also maintain deletion of the copy constructor/copy assignment operator for clarity?

Reply:
I added them back for clarity.